robotcontainer.py

```diff
@@ -121,6 +125,9 @@ def __init__(self) -> None:
         self.chooser.addOption("Complex Auto", self.complexAuto)
         self.chooser.addOption("Target Auto", self.driveToTarget)
```

Comment:
We should remove these (and the associated files) if they are no longer used, but that is not a blocker for this PR

Reply:
A lot of “remove unused” is needed through our codebase right now..